src/components/Upgrade/content.js

```diff
@@ -3,17 +3,21 @@ import large2 from './highlights/large/2.png'
 import large3 from './highlights/large/3.png'
 import large4 from './highlights/large/4.png'
 import large5 from './highlights/large/5.png'
+import large6 from './highlights/large/6.png'
+import large7 from './highlights/large/7.png'
 
 import small1 from './highlights/small/1.png'
 import small2 from './highlights/small/2.png'
 import small3 from './highlights/small/3.png'
 import small4 from './highlights/small/4.png'
 import small5 from './highlights/small/5.png'
+import small6 from './highlights/small/6.png'
+import small7 from './highlights/small/7.png'
```

Comment:
Are the current small images just copies of the large images?

Reply:
First run I exported the images at `3x` way too much, then I exported them at `1x` and optimized the files, too fuzzy, the current files were exported at `2x` and optimized.

Reply:
Oh, and sorry, to answer your question: yes.